.expeditor/verify_private.pipeline.yml

```diff
@@ -114,12 +114,15 @@ steps:
             "/go/src/github.com/chef/automate"
           ]
 
-  - label: ":semgrep: Published"
+  - label: ":semgrep: Security"
     command:
     - echo "running in $(pwd)"
     - export SEMGREP_JOB_URL=$BUILDKITE_BUILD_URL
     - export SEMGREP_BRANCH=$BUILDKITE_BRANCH
-    - python -m semgrep_agent --publish-token "\$SEMGREP_TOKEN" --publish-deployment \$SEMGREP_ID
+    - export SEMGREP_REPO_NAME=chef/automate
```

Comment:
This env var corrects links on the web UI dashboard to properly connect to our github repository.